Makefile

```diff
@@ -27,4 +27,19 @@ all-tests:
 generate build clean:
 	go $@ $(EXTRAGOARGS)
 
+sandbox-test-fc-build:
+	docker build -f fctesting/sandbox/Dockerfile -t "firecracker" .
+
+sandbox-test-fc-run:
+	docker run \
+		--init \
+		--rm \
+		--privileged \
+		--security-opt seccomp=unconfined \
+		--ulimit core=0 \
+		--device=/dev/kvm:/dev/kvm \
```

Comment:
Are you sure this `--device` is needed? I just learned recently that when you use `--privileged`, Docker will automatically create all the device nodes present under the host's `/dev` inside the container ([docs](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities))

Reply:
@sipsma - No idea, I copied this from firecracker :D. I can remove it and see if it still works.